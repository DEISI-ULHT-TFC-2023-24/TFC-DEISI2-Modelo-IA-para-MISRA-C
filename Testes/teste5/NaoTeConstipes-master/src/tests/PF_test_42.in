confadd 5
confboard
confstart
s
