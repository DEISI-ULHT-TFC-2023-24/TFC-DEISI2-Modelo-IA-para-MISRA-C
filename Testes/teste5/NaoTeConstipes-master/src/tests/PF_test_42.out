+--------------------------------------------------------+
|         Nao Te Constipes - configs                     |
+--------------------------------------------------------+
| confadd <numero>    - adicionar casa segura            |
| confsave <nome>     - gravar ficheiro de casas seguras |
| confboard <numero>  - mostrar o tabuleiro              |
| confstart <numero>  - iniciar o jogo                   |
| confmenu            - imprimir menu                    |
+--------------------------------------------------------+
>>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      || **** ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
| abcd |                                        |      |
| **** |                                        | **** |
|      |                                        | wxyz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      || **** ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
| abcd |                                        |      |
| **** |                                        | **** |
|      |                                        | wxyz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+------------------------------------+
|         Nao Te Constipes           |
+------------------------------------+
| <id do peao> (abcd, xyzw)          |
| s - sair                           |
| h - imprimir menu                  |
+------------------------------------+
------ Jogador 1 ------
Jogador lancou dados com valor 8 
>Fim do jogo
